ext/snmp/snmp.c

```diff
@@ -626,6 +626,40 @@ static void php_snmp_internal(INTERNAL_FUNCTION_PARAMETERS, int st,
 }
 /* }}} */
 
+static void php_snmp_zend_string_release_from_char_pointer(char *ptr) {
+	if (ptr) {
+		zend_string *pptr = (zend_string *)(ptr - XtOffsetOf(zend_string, val));
+		if (GC_REFCOUNT(pptr)) {
+			zend_string_release(pptr);
+		}
+	}
+}
+
+static void php_free_objid_query(struct objid_query *objid_query, HashTable* oid_ht, zend_string *value_str, HashTable *value_ht, int st) {
```

Comment:
I think it's a good idea to make value_ht and value_str const to indicate to the programmer that they are just used for checking.